src/fides/api/service/connectors/scylla_connector.py

```diff
@@ -38,41 +49,65 @@ def create_client(self) -> Cluster:
             cluster = Cluster(
                 [config.host], port=config.port, auth_provider=auth_provider
             )
-        except cassandra.UnresolvableContactPoints:
+        except UnresolvableContactPoints:
             raise ConnectionException("No host available.")
 
         return cluster
 
-    def query_config(self, node: ExecutionNode) -> QueryConfig[Any]:
-        pass
+    def query_config(self, node: ExecutionNode) -> ScyllaDBQueryConfig:
+        return ScyllaDBQueryConfig(node)
+
+    def set_schema(self, connection: Session) -> None:
+        """Optionally override to set the schema for a given cluster that
+        persists through the entire session"""
+        config = self.get_config()
+        if not config.keyspace:
+            raise ScyllaConnectorMissingKeyspace(
+                f"No keyspace provided in the ScyllaDB configuration for connector {self.configuration.name}"
```

Comment:
We don't prompt the user for a name when configuring an integration so the `name` won't be available
<img width="557" alt="image" src="https://github.com/user-attachments/assets/06b22861-4d62-4bbd-a0b3-0e7bd6620084">
It'd be more reliable to use `configuration.key`